spec/models/pixelz/modification_request_spec.rb

```diff
@@ -46,4 +46,25 @@
       end
     end
   end
+
+  describe '#reject' do
+    before do
+      @image = create(:image)
+      @modify_image_request = Pixelz::ModificationRequest.modify(@image) 
+    end
+    context 'with comment' do
+      it 'returns no error code' do
+        comment = "The top of picture still have background colour."     
```

Comment:
User single quote whenever possible if there is no string interpolation. This helps the entire app consistency. Below codes as well, `res["RejectImageResponse"]["ErrorCode"] , comment = "", etc`